src/app/core/data/root-data.service.ts

```diff
@@ -25,21 +24,21 @@ export class RootDataService extends BaseDataService<Root> {
     protected rdbService: RemoteDataBuildService,
     protected objectCache: ObjectCacheService,
     protected halService: HALEndpointService,
-    protected restService: DspaceRestService,
   ) {
-    super('', requestService, rdbService, objectCache, halService, 6 * 60 * 60 * 1000);
+    super('', requestService, rdbService, objectCache, halService, 60 * 1000);
```

Comment:
Do I understand correctly that the cached response now invalidates in 1 minute instead of 6 hours?

Reply:
Good catch! 

You're right: when I started looking into this issue, I was thinking about simply lowering the validity of that root endpoint request, but then I figured we'd still need something to do the job the guard is doing now (redirect when that request fails) so I ended up fixing the guard instead. 

I'll roll back this change to ensure it doesn't break anything unexpectedly